### ISSUE TITLE
[Feature Request] Add Visual Studio Code Server Integration

### ISSUE THREAD

**Issue by jaddel** [th33xitus/kiauh#328]
### Is your feature request related to a problem? Please describe

As a Klipper user, I often find myself needing to edit configuration files on my Raspberry Pi or other single-board computer (SBC). However, editing files directly on the SBC can be cumbersome and time-consuming. Additionally, some users may prefer to use a specific code editor or IDE, such as Visual Studio Code (VS Code).

### Describe the solution you'd like

Integrating VS Code Server into Kiauh would streamline editing of configuration files by allowing users to access and edit them remotely through a web browser, using the vs code editor.

### Describe alternatives you've considered

Connecting from my local VS Code instance via ssh.

**Comment by Frix-x** [comment]
This is already possible without the need of KIAUH: just install the [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) in VS code and you will be good to go!

Here is an example on my Windows computer connected to my Pi: there is all the folder tree on the left and the terminal on the bottom, etc...

![image](https://user-images.githubusercontent.com/10575621/232290034-1fbb549d-722c-4b66-a088-01d09e39755a.png)

**Comment by jaddel** [comment]
> This is already possible without the need of KIAUH: just install the [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) in VS code and you will be good to go!
> 
> Here is an example on my Windows computer connected to my Pi: there is all the folder tree on the left and the terminal on the bottom, etc...
> 
> ![image](https://user-images.githubusercontent.com/10575621/232290259-1e2055ed-fea3-440a-8a19-b9f092f3e4e9.png)

Hi, thanks for suggesting using a local isntall via ssh. It's actually already stated as alternative solution, but I think it would still make sense to integrate code server so all this could be done via browser with no local install needed.

**Comment by th33xitus** [comment]
To be frank, im not quite sure if this is something that should be in the scope of KIAUH. I assume most people use Mainsail or Fluidd to edit their configurations and if they edit other config files which are not accessible through one of the frontends, they do it via SSH anyways.

It doesn't seem to be a cumbersome process to install it, by looking at the repo here: <https://github.com/coder/code-server#getting-started> (hopefully i went to the right repo? i never used this software ever before)
So for anybody really looking to use that, following those 4 steps should be no issue.

So basically it would take way more time and effort for me to write the corresponding wrapper for it in KIAUH and then test everything so it works, for something that i don't really consider should be part of KIAUHs scope anyways.
